tests/unit/CoreListTest.php

```diff
@@ -0,0 +1,373 @@
+<?php
+
+namespace WPGraphQL\ContentBlocks\Unit;
+
+final class CoreListTest extends PluginTestCase {
+	/**
+	 * The ID of the post created for the test.
+	 *
+	 * @var int
+	 */
+	public $post_id;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->post_id = wp_insert_post(
+			[
+				'post_title'   => 'Post with List',
+				'post_content' => '',
+				'post_status'  => 'publish',
+			]
+		);
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function tearDown(): void {
+		parent::tearDown();
+
+		wp_delete_post( $this->post_id, true );
+
+		\WPGraphQL::clear_schema();
+	}
+
+	public function query(): string {
```

Comment:
We should also be querying/testing the `CoreListItemAttributes`

Reply:
In _this_ class, not as a separate one.

Bring over the fragment and include it in the query, change `editorBlocks {` to `editorBlocks( flat: false) {` and then test the `innerBlocks` aka various our list items.

Reply:
(And make sure to fix the formatting when you bring stuff over)

Reply:
Done ✅